doc/website/getting-started/list-of-examples.md

```diff
@@ -0,0 +1,16 @@
+# List of examples
+
+| Example                                         | Description                                                 | Level              |
+|:------------------------------------------------|:------------------------------------------------------------|:-------------------|
+|[icehello](./examples/icehello/)                 | How to send data to another process                         | :star:             |
+|[icedelivery](./examples/icedelivery/)           | How to send and receive data in a functional way            | :star:             |
+|[iceoptions](./examples/iceoptions/)             | Configure the history cache size and the starting behaviour | :star:             |
+|[icedelivery_in_c](./examples/icedelivery_in_c/) | How to send and receive data using the C API                | :star:             |
+|[callbacks](./examples/callbacks/)               | Implement callbacks which are triggered by events           | :star::star:       |
```

Comment:
Implement event triggered callbacks.

Reply:
Those match the titles in the other file I think so the comments there apply here as well.